tools/SourceKit/lib/SwiftLang/SwiftEditor.cpp

```diff
@@ -963,6 +963,17 @@ class SemanticAnnotator : public SourceEntityWalker {
     if (AvailableAttr::isUnavailable(D))
       return true;
 
+    auto &SM = D->getASTContext().SourceMgr;
+    if (D == D->getASTContext().getOptionalNoneDecl() &&
+        SM.extractText(Range) == "nil") {
```

Comment:
```suggestion
        SM.extractText(Range, BufferID) == "nil") {
```
So it skips `findBufferContainingLoc()` operation

Reply:
Good catch. Thanks 👍